lib/ansible_tower_client/base_models/inventory_source.rb

```diff
@@ -4,6 +4,10 @@ def self.endpoint
       "inventory_sources".freeze
     end
 
+    def self.create(_api, _attributes)
+      "POST not allowed"
```

Comment:
@Fryguy - it should probably raise an error - Making a direct call to the API does.  I'll get that changed.